Assignment 2/Constant.java

```diff
@@ -0,0 +1,216 @@
+
+public class Constant {
+
+    final String Enter_String = "Enter the first String - ";
+    final String Entry = "You have Entered - ";
+    final String Operation = "Choose which operation you want to perform.";
+    final String Append = "1. Append ";
+    final String Count = "2. Count words ";
+    final String Replace = "3. Replace character ";
+    final String Palindrome = "4. Palindrome ";
+    final String Splice = "5. Splice ";
+    final String Split = "6. split ";
+    final String Repeat = "7. Repeat ";
+    final String Sort = "8. Sort ";
+    final String Shift = "9. Shift ";
+    final String Reverse = "10. Reverse ";
+    final String Exit = "0. Exit ";
+    final String Choice = "Enter The number of The operation you want to perform - ";
+    final String Invalid = "Invalid Input ! \nPlease enter Number between 1 to 10.";
+    final String Enter_Append = "Enter a string you want to append: ";
+    final String Output = "Your output is: ";
+    final String Input = "Enter your string: ";
+    final String End = "Now please rate me 1 to 5.";
+    final String NewString = "11. New Input String";
+    final String startIndex = "Enter start index(int only): ";
+    final String endIndex = "Enter length(int only): ";
+    final String SplitInput = "Enter the character(char only): ";
+    final String OldChar = "Enter the old character(char only): ";
+    final String NewChar = "Enter the new character(char only): ";
+    final String rotateInt = "Enter the number(int only): ";
+
+    public static void main(String[] args) {
+        
+    }
+    
+    // Function to append a string

```

Comment:
Always mention the parameters and return -> what does this method is used for that is the parameter and what will be the return value -> mention them properly